defopt.py

```diff
@@ -158,7 +166,9 @@ class _NoTypeFormatter(_Formatter):
     show_types = False
 
 
-def _populate_parser(func, parser, parsers, short):
+def _populate_parser(func, parser, parsers, **kwargs):
+    short = kwargs['short']
+    strict_kwonly = kwargs['strict_kwonly']
```

Comment:
I prefer unpacking arguments only when they are actually needed, in order to avoid explicitly forwarding long argument lists throughout the call stack, but here with just two arguments either way would be reasonable; let me know if you want this to be changed back.